tests/rules/test_braces.py

```diff
@@ -31,6 +31,24 @@ def test_disabled(self):
                    'dict6: {  a: 1, b, c: 3 }\n'
                    'dict7: {   a: 1, b, c: 3 }\n', conf)
 
+    def test_forbid(self):
+        conf = ('braces:\n'
+                '  forbid: false\n')
+        self.check('---\n'
+                   'dict: {}\n', conf)
+        self.check('---\n'
+                   'dict: {a: 1}\n', conf)
```

Comment:
Can you add these examples please?
```yaml
dict: {a}
```
```yaml
dict: {
  a: 1
}
```
(same with `forbid: true` and with flow sequences)

Reply:
Thanks for your suggestion! I applied it, squashed and force-pushed.

Reply:
Thanks, but you didn't add the same to `forbid: true` and for flow sequences. It would be useful to check these:
```python
        self.check('---\n'
                   'dict: {a}\n', conf, problem=(2, 8))
```
and:
```yaml
array: [
  a,
  b,
]
```
↑ both with `forbid: true` and `false`

Reply:
Thanks! I added these test cases also.